app/controllers/sign_up/completions_controller.rb

```diff
@@ -124,6 +125,11 @@ def dob
       pii_dob ? pii_dob.to_date.to_formatted_s(:long) : ''
     end
 
+    def verified_at
+      timestamp = current_user.active_profile&.verified_at
```

Comment:
What does the `&` do here?

Reply:
That's the nil-safe accessor, same as `active_profile.try(:verified_at)` that Rails monkeypatches: http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/

Reply:
Oh wow! TIL.  Thanks!